docs/authentication/social-connections/oauth.mdx

```diff
@@ -69,13 +69,13 @@ Using these tokens, you can query the respective OAuth providers for additional
   Clerk ensures that the OAuth Access Token will be always fresh so that you don't have to worry about OAuth Refresh Tokens anymore.
 </Callout>
 
-## Add Social Connection after Sign Up
+## Add Social Connection after sign-up
```

Comment:
Hyphenated "sign-up" is a noun correct? Don't we still capitalize here?

Reply:
yes, you are correct. why would we capitalize? we use sentence-casing for titles. We capitalize Social Connection throughout our docs, so that capitalization is correct.

Reply:
I typically recommend removing the hyphen ("frontend, website, signup"), as eventually all these words become concatenated. Seems correct to not be capitalized, but I don't know the existing convention with "Social Connection?"

Reply:
This makes sense. We'll add it to the style guide. Might want to add a little bit of visual flavor to Clerk Terminology/link them to a glossary somewhere. Approved!